cmd/podman/common/completion.go

```diff
@@ -22,62 +23,109 @@ var (
 	LogLevels = []string{"debug", "info", "warn", "error", "fatal", "panic"}
 )
 
-func getContainers(status string, toComplete string) ([]string, cobra.ShellCompDirective) {
+func getContainers(toComplete string, statuses ...string) ([]string, cobra.ShellCompDirective) {
 	suggestions := []string{}
 	listOpts := entities.ContainerListOptions{
 		Filters: make(map[string][]string),
 	}
 	listOpts.All = true
+	listOpts.Pod = true
 
-	if status != "all" {
-		listOpts.Filters = map[string][]string{"status": {status}}
+	/* 	The api doesn't handle several different statuses correct see:
+		https://github.com/containers/podman/issues/8344
+
+	if len(status) > 0 {
+		listOpts.Filters["status"] = statuses
```

Comment:
I'm a little uncomfortable with a commented-out code block of this size: first, it's syntactically wrong (`status` should be `statuses`); second, it may grow even staler as time goes by. I appreciate the intention, but would it be better spelled out as a more generic TODO comment instead? (Same applies below)